infra/scripts/azure-runner.sh

```diff
@@ -0,0 +1,158 @@
+    kubectl create secret generic feast-postgresql --from-literal=postgresql-password="$PG_PASSWORD" -n "$NAMESPACE"
+}
+
+function helm_install {
+    # helm install Feast into k8s cluster and display a nice error if it fails.
+    # Usage: helm_install $RELEASE $DOCKER_REPOSITORY $GIT_TAG ...
+    # Args:
+    #   $RELEASE is helm release name
+    #   $DOCKER_REPOSITORY is the docker repo containing feast images tagged with $GIT_TAG
+    #   ... you can pass additional args to this function that are passed on to helm install
+
+    local RELEASE=$1
+    local DOCKER_REPOSITORY=$2
+    local GIT_TAG=$3
+
+    shift 3
+
+    # We skip statsd exporter and other metrics stuff since we're not using it anyway, and it
+    # has some issues with unbound PVCs (that cause kubectl delete pvc to hang).
+    echo "${STEP_BREADCRUMB} Helm installing feast"
+
+    if ! time helm install --wait "$RELEASE" infra/charts/feast \
+        --timeout 15m \
+        --set "feast-jupyter.image.repository=${DOCKER_REPOSITORY}/feast-jupyter" \
+        --set "feast-jupyter.image.tag=${GIT_TAG}" \
+        --set "feast-online-serving.image.repository=${DOCKER_REPOSITORY}/feast-serving" \
+        --set "feast-online-serving.image.tag=${GIT_TAG}" \
+        --set "feast-jobservice.image.repository=${DOCKER_REPOSITORY}/feast-jobservice" \
+        --set "feast-jobservice.image.tag=${GIT_TAG}" \
+        --set "feast-core.image.repository=${DOCKER_REPOSITORY}/feast-core" \
+        --set "feast-core.image.tag=${GIT_TAG}" \
+        --set "prometheus-statsd-exporter.enabled=false" \
+        --set "prometheus.enabled=false" \
+        --set "grafana.enabled=false" \
+        --set "feast-jobservice.enabled=false" \
+        "$@" ; then
+
+        echo "Error during helm install. "
+        kubectl -n "$NAMESPACE" get pods
+
+        readarray -t CRASHED_PODS < <(kubectl -n "$NAMESPACE" get pods --no-headers=true | grep "$RELEASE" | awk '{if ($2 == "0/1") { print $1 } }')
+        echo "Crashed pods: ${CRASHED_PODS[*]}"
+
+        for POD in "${CRASHED_PODS[@]}"; do
+            echo "Logs from pod error $POD:"
+            kubectl -n "$NAMESPACE" logs "$POD" --previous
+        done
+
+        exit 1
+    fi
+}
+
+function setup_sparkop_role {
+    # Set up permissions for the default user in sparkop namespace so that Feast SDK can manage
+    # sparkapplication resources from the test runner pod.
+
+    cat <<EOF | kubectl apply -f -
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: use-spark-operator
+rules:
+- apiGroups: ["sparkoperator.k8s.io"]
+  resources: ["sparkapplications"]
+  verbs: ["create", "delete", "deletecollection", "get", "list", "update", "watch", "patch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  name: use-spark-operator
+roleRef:
+  kind: Role
+  name: use-spark-operator
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: default
+EOF
+}
```

Comment:
it may make sense to put these functions in a separate file since they're used by the AWS test scripts as well